### PR TITLE
feat: add packaging for pypi

### PR DESCRIPTION
**Pull request by pythonbrad**
### Add a configuration for the deployment on pypi.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        long_description=LONG_DESCRIPTION,
+        long_description_content_type="text/markdown",
+        packages=find_packages(where="src"),
+        install_requires=["urllib3>=1.26.8<2.0"],
+        keywords=["link", "url", "broken", "check"],
+        classifiers=[
+            "Topic :: Internet :: WWW/HTTP",
+        ],
+        python_requires='>=3.9',
+        package_dir={"": "src"},
```
Comment (Sanix-Darker) [inline setup.py:26]: There is an error with this line it think 🤔 
If am not,, wrong, there is no 'src' directory only 'broken_link_checker'

**Review by Sanix-Darker — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Setting up
+setup(
+        name="broken_link_checker",
+        version=VERSION,
+        author="pythonbrad",
+        author_email="fomegnemeudje@outlook.com",
+        description=DESCRIPTION,
+        long_description=LONG_DESCRIPTION,
+        long_description_content_type="text/markdown",
+        packages=find_packages(where="src"),
```
Comment (Sanix-Darker) [inline setup.py:19]: same remark here !

File: setup.py (hunk 1 above)
Comment (pythonbrad) [reply setup.py:26]: the broken_link_folder is inside the src folder

**Review by Sanix-Darker — APPROVED** [review]
Sound good for me then !
Go for a squash and merge !

**Pull request merged**